content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
 
 +++
 
+## What is Kubeflow?
+
 Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
 Operators can choose what is best for their users, there is no requirement to deploy every component.
-To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
+Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
```

Comment:
I think we can put a newline here for visual separation:

```suggestion

Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
```

Reply:
This seems not to be addressed.

Reply:
This is in reply to a suggestion for a newline.

It was added a few commits ago, I can see it on the test website:
- https://deploy-preview-3693--competent-brattain-de2d6d.netlify.app/docs/started/installing-kubeflow/

Reply:
SGTM